pdfminer/layout.py

```diff
@@ -598,27 +738,61 @@ def group_objects(self, laparams, objs):
         return
 
     # group_textlines: group neighboring lines to textboxes.
-    def group_textlines(self, laparams, lines):
+    def group_textlines(self, laparams, lines, splitobjs=[]):
+        ''' group_textlines: group neighboring lines to textboxes
+
+        # ARGUMENTS
+            laparams: laparams obj -- an object containing all the parameters
+                needed to join together characters and make lines of text
+            lines: list -- each member is an LTLayoutContainer object
+                containing a unique textline
+            splitobjs: list -- each member is an LTRect object
+        '''
         plane = Plane(self.bbox)
         plane.extend(lines)
         boxes = {}
+        # iterate through all neighbors and join any that are text that
+        # pass requirements
         for line in lines:
             neighbors = line.find_neighbors(plane, laparams.line_margin)
             if line not in neighbors:
                 continue
             members = []
             for obj1 in neighbors:
+                # for each text line to be merged, see if it is split by
+                # an LTRect object
+                if line is not obj1 and splitobjs:
+                    if self.vsplitted(line,
+                                      obj1,
+                                      splitobjs,
+                                      laparams.cell_margin):
+                        continue
```

Comment:
Use if-else instead of `continue`

Reply:
Sorry for the delay.  COVID has completely interrupted my standard schedule, so this is the first time getting back to this.

I have just looked this over and this will be a big hurdle for me to address.  I am worried that this request will make the logic much more difficult to parse.  I'll look at it again tomorrow.

Reply:
If you want, I can try and do it myself?

Reply:
I went ahead and replaced the continue with a flag.  What are your thoughts on this solution?